app/controllers/users_controller.rb

```diff
@@ -1,6 +1,10 @@
 class UsersController < ApplicationController
+  before_action :authenticate_user!
   def index
-    @users = User.all
+    @users = User.where.not("id = ?",current_user.id).
+    order("created_at DESC")
+    @conversations = Conversation.involving(current_user).
+    order("created_at DESC")
```

Comment:
Use 2 (not 0) spaces for indenting an expression in an assignment spanning multiple lines.